config/betterquesting/DefaultQuests/Quests/Tier7ZPM-AAAAAAAAAAAAAAAAAAAAHg==/RecycleTrinium-AAAAAAAAAAAAAAAAAAALgg==.json

```diff
@@ -29,7 +29,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "You can get the Trinium dust by reacting it with Hydrogen in LCR or from electrolysis."
+      "desc:8": "You can get the Trinium dust by reacting it with Hydrogen in a LCR."
```

Comment:
**an** LCR.

Reply:
I actually thought about it but was like "no the first word is Large so it should be **a**". But because it's an "el" sound it is correct to be **an**. Fixing...